src/style/panel.less

```diff
@@ -455,3 +455,9 @@
         min-height: 3rem;
     }
 }
+
+#reference-none {
```

Comment:
I could be wrong, but _generally_ we do not use CSS ids for styling. Perhaps @designedbyscience and @placegraphichere have carved out some exceptions for SVG. But, I would have expected a class selector instead.